idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/settings/tabs/export/ImageVectorExportSettingsScreen.kt

```diff
@@ -93,6 +95,10 @@ private fun ImageVectorExportSettingsUi(
             checked = generatePreview,
             onCheckedChange = { onAction(UpdatePreviewGeneration(it)) },
         )
+        IndentSizeSection(
+            indent = indentSize,
+            onValueChange = { onAction(SettingsAction.UpdateindentSize(it)) },
```

Comment:
`SettingsAction` can be imported

<img width="1201" alt="Screenshot 2024-11-05 at 12 45 49" src="https://github.com/user-attachments/assets/e648d125-7709-4f16-8379-220648ca9a45">